source/faq.txt

```diff
@@ -115,3 +115,9 @@ If you see slow operation warnings, check CPU, memory, and network usage on the
 source and destination clusters.  If the clusters are underprovisioned for your
 needs, consider upgrading the cluster hardware.
 
+Does ``mongosync`` Support TTL Indexes?
+----------------------------------------
+
+{+c2c-product-name+} supports syncing :ref:`index-feature-ttl`.
```

Comment:
**Question**

Does this mean that the TTL index itself will be synced from Deployment A -> Deployment B or that the data which is expired by the TTL index will synced? I suspect the former, but a bit more detail indicating this would be helpful.

Reply:
Fixed. Per Ajay, both are true but we only want to emphasize the former in this context.